### PR TITLE
Feat: Add form to create new user from ui, Fix issue-3480 [UI] Default Value for User's not Assigned to any Teams on User's Listing Page

### LINKED ISSUE #2932: Add a User through Users page
**Affected module**
UI, Ingestion
**Describe the bug**
Helm chart based cluster doesn't ingest any users data. This means there are no users 
and there is no way to create ingestion without an active user

**To Reproduce**

Screenshots or steps to reproduce

**Expected behavior**

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### LINKED ISSUE #3480: [UI] Default Value for User's not Assigned to any Teams on User's Listing Page
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
UI

**Describe the bug**
Let's add a default value when user is not assigned to any Team. Ex. `None`.
image

**Version:**
 - OpenMetadata version: 0.9.0

cc - @darth-coder00, @harshach

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Closes #2932 
I have added the form to create new user from ui, it is accessible only to admin.
admin can able to add new user from users route.
Closes #3480 
Fixed issue-3480 [UI] Default Value for User's not Assigned to any Teams on User's Listing Page

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :
image

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@harshach @vivekratnavel @darth-coder00 @Sachin-chaurasiya @shahsank3t

### CODE REVIEW

**Comment by darth-coder00** [comment]
@ShaileshParmar11 Please fix failing unit tests

File: openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+          data-testid="email"
+          id="email"
+          name="email"
+          placeholder="email"
+          type="text"
+          value={email}
+          onChange={handleValidation}
+        />
+        {showErrorMsg.email && errorMsg('Email is required.')}
+        {showErrorMsg.validEmail && errorMsg('Email is invalid.')}
```
Comment (darth-coder00) [inline openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx:254]: This should be like,
```
{showErrorMsg.email ? errorMsg('Email is required.') 
 : showErrorMsg.validEmail ? errorMsg('Email is invalid.') : null}
```

Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx:254]: Also error messages should be from `en.ts`, under json key `form-error-messages` similar to `api-error-messages`

File: openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+      <Field className="tw-flex tw-gap-5">
+        <div className="tw-flex tw-pt-1">
+          <label>Admin</label>
+          <div
+            className={classNames('toggle-switch', { open: isAdmin })}
+            data-testid="admin"
+            onClick={() => {
+              allowAccess && setIsAdmin((prev) => !prev);
+            }}>
```
Comment (darth-coder00) [inline openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx:299]: This should turn the `isBot` state to `false`

File: openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+          </div>
+        </div>
+        <div className="tw-flex tw-pt-1">
+          <label>Bot</label>
+          <div
+            className={classNames('toggle-switch', { open: isBot })}
+            data-testid="bot"
+            onClick={() => {
+              allowAccess && setIsBot((prev) => !prev);
+            }}>
```
Comment (darth-coder00) [inline openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx:310]: This should turn the `isAdmin` state to `false`

File: openmetadata-ui/src/main/resources/ui/src/jsons/en.ts [hunk 4]
```diff
@@ -15,6 +15,7 @@ const jsonData = {
   'api-error-messages': {
     'add-glossary-error': 'Error while adding glossary!',
     'add-glossary-term-error': 'Error while adding glossary term!',
+    'add-user-error': 'Error while adding user!',
```
Comment (darth-coder00) [inline openmetadata-ui/src/main/resources/ui/src/jsons/en.ts:18]:
```suggestion
    'create-user-error': 'Error while creating user!',
```

**Review by darth-coder00 — CHANGES_REQUESTED** [review]
Few changes needed

File: openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/CreateUser/CreateUser.component.tsx:299]: Totally forgot about this 😅, great catch!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by harshach — APPROVED** [review]

**Pull request merged**